spec/requests/user_profile_spec.rb

```diff
@@ -81,14 +81,6 @@
     end
   end
 
-  describe "GET /user" do
```

Comment:
I'm not sure I understand why this test should be removed in support of the change in `users/show`, would you mind clarifying it (just for my benefit, if nothing else)?

Reply:
Sorry, forgot to explain it's a duplicate of the same test on line 8 https://github.com/thepracticaldev/dev.to/blob/620b72ce4be52cdf694d0fde893c190ee4e44746/spec/requests/user_profile_spec.rb#L8